src/js/components/Pagination/__tests__/__snapshots__/Pagination-test.tsx.snap

```diff
@@ -1596,7 +1596,8 @@ exports[`Pagination should apply button kind style when referenced by a string 1
   border-width: 2px;
   border-color: skyblue;
   padding: 2px 20px;
-  background: #33333310;
+  background-color: #33333310;
+  color: #000000;
   background-color: rgba(202,156,234,1);
```

Comment:
Something looks a little strange here. We seem to be setting `background-color` and `color` twice each.

Reply:
Yes, I agree that this behavior isn't great. While I was working on this it felt like the entire way we are setting background and colors in `background.js` needs an overhaul. If you look at the snapshots prior to the changes made in this PR, there are lots places where we set background and color multiple times within the same css class (for an example look at the snapshot for 'Button kind primary button 1' class 'c3' another good place to look is at the Pagination snapshots). I wasn't sure if we want to handle that in this PR or separately?

Reply:
OK. Thanks for the explanation.